gms/client/src/main/java/com/linkedin/common/client/DatasetsClient.java

```diff
@@ -18,9 +18,4 @@ protected DatasetKey toDatasetKey(@Nonnull DatasetUrn urn) {
                 .setOrigin(urn.getOriginEntity())
                 .setPlatform(urn.getPlatformEntity());
     }
-
-    @Nonnull
-    protected DatasetUrn toDatasetUrn(@Nonnull DatasetKey key) {
-        return new DatasetUrn(key.getPlatform(), key.getName(), key.getOrigin());
-    }
 }
```

Comment:
Probably rename this as BaseDatasetsClient?

Reply:
I will prefer to move the method `toDatasetKey` in `DatasetsClient` to a utility file, will leave it for another PR.